CONTRIBUTING.md

```diff
@@ -54,6 +54,16 @@ mvn clean verify -Pnative
 # Wait... success!
 ```
 
+## Build with Gradle
+
+Clone like above, and build e.g. `getting-started` using:
+
+    ../gradlew build
+
+You'll find the `*-runner.jar` in `build` (not `build/libs`).
+
+We may change it to run on the regular "build" task instead of "build-shamrock" later.
```

Comment:
The `build` task automatically runs `quarkusBuild` so this can be removed I think.

Reply:
Oups, yeah, of course, well spotted.. line 61 used to be `gradlew quarkusBuild` and I changed but didn't remove this.